runtime/src/main/java/org/corfudb/runtime/view/stream/AbstractQueuedStreamView.java

```diff
@@ -64,27 +66,39 @@ protected void addToResolvedQueue(QueuedStreamContext context,
     @Override
     protected ILogData getNextEntry(QueuedStreamContext context,
                                     long maxGlobal) {
+        NavigableSet<Long> pollFrom;
```

Comment:
See 9 lines down, line 78: "Is there checkpoint data to consume first?"

I added that variable to avoid checking repeatedly if context.readCpQueue is empty.